use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -57,19 +58,13 @@ This storage type gives you up to 64 TB and 32,000 IOPS, and is available under
 1. **In [$CONSOLE][console], select your $SERVICE_SHORT, then click `Operations` > `Compute and storage`**
 1. **Select `Enhanced` in the `Storage type` dropdown**
 
-    ![Enhanced storage in Timescale Cloud](https://assets.timescale.com/docs/images/enhanced-storage-timescale-cloud.png)
-
-    <Highlight type="note">
-   
-    The enhanced storage is currently only available in `us-east-1` with plans to extend to other regions. 
```

Comment:
AWS doesn't offer io2 in Sao Paulo, so that's the only region where we can't offer Enhanced Storage https://iobeam.slack.com/archives/C08BD2XC4J1/p1747762919639789?thread_ts=1747635358.981409&cid=C08BD2XC4J1

Reply:
updated